glyphs-reader/src/lib.rs

```diff
@@ -6,7 +6,7 @@ mod from_plist;
 mod plist;
 mod to_plist;
 
-pub use font::{Axis, Component, Font, FontMaster, Glyph, Layer, Node, NodeType, Path};
+pub use font::{Axis, Component, Font, FontMaster, Layer, Node, NodeType, Path, RawGlyph};
```

Comment:
why pub use font::RawGlyph instead of font::Glyph?

Reply:
Oops, I think that's because I did an automated rename.

Reply:
Yup, ended up with RawLayer there too. Fixed.